src/v/redpanda/admin_server.cc

```diff
@@ -1318,6 +1320,106 @@ void admin_server::register_partition_routes() {
             });
       });
 
+    /*
+     * Abort transaction for partition
+     */
+    ss::httpd::partition_json::abort_transaction.set(
+      _server._routes,
+      [this](std::unique_ptr<ss::httpd::request> req)
+        -> ss::future<ss::json::json_return_type> {
+          auto ns = model::ns(req->param["namespace"]);
```

Comment:
hm, I think it is time to refactor parsing ntp from request for all partition requests

Reply:
great idea. we do this a lot!